src/Sulu/Bundle/MediaBundle/Search/Subscriber/MediaSearchSubscriber.php

```diff
@@ -93,10 +93,9 @@ public static function getSubscribedEvents()
     public function handlePreIndex(PreIndexEvent $event)
     {
         $metadata = $event->getMetadata();
-        $reflection = $metadata->getClassMetadata()->reflection;
 
         if (
-            false === $reflection->isSubclassOf(FileVersionMeta::class)
+            !$event->getSubject() instanceof FileVersionMeta
             && FileVersionMeta::class !== $metadata->getName()) {
```

Comment:
Can you move the closing bracket from the if into a separate line? This way it reads a little bit strange.